### PR TITLE
SD-2191-CS non empty response check added

### PR DESCRIPTION
**Pull request by palatsangeetha**

### CODE REVIEW

File: commercial-schedules/src/test/java/org/dcsa/conformance/standards/cs/checks/CsChecksTest.java [hunk 1]
```diff
@@ -63,4 +63,12 @@ void testValidateCutoffTimeCodePs() {
     rootNodeArray.add(schedule);
     assertFalse(VALIDATE_CUTOFF_TIME_CODE_PS.validate(rootNodeArray).isEmpty());
   }
+
+  @Test
+  void testValidateNonEmptyResponse() {
+    rootNodeArray.add(objectMapper.createObjectNode());
+    assertTrue(CsChecks.VALIDATE_NON_EMPTY_RESPONSE.validate(rootNodeArray).isEmpty());
+    rootNodeArray.removeAll();
+    assertFalse(CsChecks.VALIDATE_NON_EMPTY_RESPONSE.validate(rootNodeArray).isEmpty());
```
[the remark this answers is not in the document]
Comment (palatsangeetha) [reply commercial-schedules/src/test/java/org/dcsa/conformance/standards/cs/checks/CsChecksTest.java:72]: > Should we also validate the expected error message in the case where it exists instead of just asserting if its empty?
> 
> Also I was used to see one test method only testing 1 condition, so for this case we would have 2 tests one for when the response is empty and one for when it's not. However I've seen that you have always been adding the tests in the same method and maybe it doesn't make sense to do differently now.

Thanks for the input, @pedrocarvalhodcsa. Since the exact error message isn't relevant for this test scenario and the goal is simply to confirm that validation fails, I've opted to assert only that the result is non-empty. Please let me know if you'd like me to include a check for the specific error message as well, happy to update the test.

**Comment by palatsangeetha** [comment]
Thanks for the approval @gj0dcsa, since I don't have write access to 'test' I won't be able to merge this PR myself. Would you be able to merge on my behalf?

File: commercial-schedules/src/test/java/org/dcsa/conformance/standards/cs/checks/CsChecksTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (pedrocarvalhodcsa) [reply commercial-schedules/src/test/java/org/dcsa/conformance/standards/cs/checks/CsChecksTest.java:72]: Okay it makes sense. You don't need to add if you think it's not relevant. Approved ✅
